### PR TITLE
🌱: Add build config for iOS to make device debugging easier

### PR DESCRIPTION
**Pull request by Takumon**
## やったこと

in build config for iOS
- [x] Add a template file to set personal account information
- [x] Modified build configs to load the template
- [x] Add build config for Debug Advanced to prepare for the build switch feature.
- [x] Add description in README

## やっていないこと

Nothing

## 動作確認

- [x] Clone this brunch
- [x] Set my personal account information in the template
- [x] Connect iPhone to Mac mini
- [x] Build in Xcode and Install an app to iPhone
- [x] Launch app in iPhone

## デバイス

- [x] iOS
  - [x] シミュレータ (iPhone Xs/iOS 14)
  - [x] 実機 (iPhone 8/iOS 14)
- [ ] ~~Android~~
  - [ ] ~~エミュレータ (Pixel 3a/Android 11)~~
  - [ ] ~~実機 (Pixel 3a/Android 11)~~

## その他（レビュアーへの申し送り事項、懸念点など）

Nothing

### CODE REVIEW

**Review by in-a-paddy — CHANGES_REQUESTED** [review]

File: gitignore [hunk 1]
```diff
@@ -64,3 +64,6 @@ web-build/
 
 # Jest
 junit.xml
+
+# Local environmental variables
+ios/PersonalAccount.xcconfig
```
Comment (in-a-paddy) [inline gitignore:69]: Add this to `.gitignore`

**Review by in-a-paddy — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -31,6 +31,19 @@ npm run android    # Androidエミュレータが起動します
...
+
+1. Apple ID設定用のファイルを用意
+   ```
+   cp ios/PersonalAccount.xcconfig.template ios/PersonalAccount.xcconfig
+   ```
+2. `ios/PersonalAccount.xcconfig`にApple IDの設定を追加
+   * `CODE_SIGN_STYLE` は `Automatic` に設定
+   * `PERSONAL_IDENTIFIER` は他の人と重複しない値に設定（Bundle Identiferのサフィックスとして利用されます）
+   * `DEVELOPMENT_TEAM` は、Xcodeで個人のApple IDをTEAMとして設定したときに表示される値を設定
+
```
Comment (in-a-paddy) [inline README.md:46]: Wouldn't it be better to write it in the 'docs'?

**Review by in-a-paddy — CHANGES_REQUESTED** [review]

File: gitignore (hunk 1 above)
Comment (Takumon) [reply gitignore:69]: I have just added this to `.gitignore`.

File: README.md (hunk 2 above)
Comment (Takumon) [reply README.md]: I have just move this to `docs`

**Comment by in-a-paddy** [comment]
## Prerequisites

* Install [ios-deploy](https://github.com/ios-control/ios-deploy) (`brew install ios-deploy`, `npm i -g ios-deploy` etc.)
* Launch Xcode (`xed ios` at project root)
* Open project's "Signing & Caperbilities" and check if a signing error is not occurring

## How to run app in actual devices

* Connect iPhone to macOS
* Trust computer (in iPhone)
* `npm run ios -- --device '<device name>'`
* or `npm run ios -- --uuid <device uuid>`

**Review by in-a-paddy — APPROVED** [review]

**Comment by in-a-paddy** [comment]
🎉

**Pull request merged**